js/marquee.js

```diff
@@ -0,0 +1,28 @@
+const marqueeDataDisplayer = document.getElementById("marqueeData");
+urlForCompanyInfo =
+  "https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/";
+
+class Marquee {
+  constructor(element) {
+    this.element = element;
+  }
+  async fetchMarqueeData(urlForCompanyInfo) {
+    const response = await fetch(
+      urlForCompanyInfo + "stock-screener?/AAPL?&limit=30"
+    );
+    const data = await response.json();
+    const presentMarqueeData = document.createElement("div");
+    presentMarqueeData.classList.add("marquee", "visual-aid");
+    this.element.append(presentMarqueeData);
+
+    data.forEach((item) => {
+      let symbol = item.symbol;
+      let price = item.price;
+
+      marqueeDataDisplayer.innerHTML += `<span class="symbol-for-marquee">${symbol}</span> <span class="price-for-marquee">$${price} </span> `;
```

Comment:
DOM manipulation should be outside the loop. Search for the mentoring session which we have covered it.